string[] state_names
float64[] goal_state_values
float64[] proactive_state_values
float64[] motion_state_values 

